data_transfer/config.py

```diff
@@ -38,12 +39,11 @@ class GlobalConfig(BaseSettings):
     dreem_users: Path = csvs_path / "dreem_users.csv"
     dreem_devices: Path = csvs_path / "dreem_devices.csv"
 
-    ucam_data: Path = csvs_path / "ucam_db.csv"
-
     database_uri: str = "mongodb://user:password@localhost:27017"
     database_name: str = "pipeline_local"
 
     inventory_api: str = ""
+    ucam_api: str = ""
```

Comment:
How come this should be in the `Settings`, but `inventory_api` does not?

Reply:
@davidverweij -- you're right. Probs best to move `inventory_api` to `Settings` too 🚀

Reply:
Worth noting that the mocking of the inventory tests are not ideal because they depend on method internals e.g., `test_record_by_device_id_history_is_none` mocks `requests.get` when it should mock `device_history` (where requests.get is used) and remove the dependency on inventory_api, e.g., 

from: 
```python
    response.json = lambda: mock_inventory_history_response

    with patch("requests.get", return_value=response):
```
to
```python
    response.json = lambda: mock_inventory_history_response['data']

    with patch.object(inventory, "device_history", return_value=response):
```

Reply:
So, when I move inventory (and thus ucam as well) from the `GlobalConfig` to `Settings`, test won't run (due to the `env` TESTING="True") and the `inventory_api` and `ucam_api` is not set. Thoughts?